.buildkite/pipeline.ml.yml

```diff
@@ -153,9 +153,12 @@
     - conda create -n rllib_contrib python=3.8 -y
     - conda activate rllib_contrib
     - (cd rllib_contrib/a2c && pip install -r requirements.txt && pip install -e ".[development]")
+    - cp rllib/tests/run_regression_tests.py rllib_contrib/a2c/.
     - ./ci/env/env_info.sh
-    - pytest rllib_contrib/a2c/tests/
+    # Examples:
     - python rllib_contrib/a2c/examples/a2c_cartpole_v1.py --run-as-test
```

Comment:
should we move examples to the bazel BUILD file?

Reply:
Great idea, was thinking about this, too. Will make it cleaner. :+1:

Reply:
done